client/src/js/services/StockService.js

```diff
@@ -3,9 +3,9 @@ angular.module('bhima.services')
 
 StockService.$inject = [
```

Comment:
This service is getting complex - we should make an issue to break it up into multiple smaller services.